Error running list_test

I'm trying to use [gickup](https://github.com/cooperspencer/gickup) with my bitbucket account and was having an issue getting it to authenticate properly. I pulled this repo down and attempted to run the `tests/list_test.go` test and get the following:
`
$ env | grep BITBUCKET
BITBUCKET_TEST_REPOSLUG=drewmoseley/blink
BITBUCKET_TEST_OWNER=drewmoseley
BITBUCKET_TEST_USERNAME=drewmoseley
BITBUCKET_TEST_PASSWORD=redacted
$ go test -v ./tests/list_test.go 
=== RUN   TestList
panic: test timed out after 10m0s
<snip>
`

This is consistent with what I see using gickup so I suspect I am doing something wrong wiith the OWNER or USERNAME field or some such.  I used an app password created with all permissions. I get the same behavior whether I use `drewmoseley/blink` or `blink` for the REPOSLUG value.

Does anyone have any troubleshooting suggestions here?

--- a/client.go
+++ b/client.go
@@ -320,13 +320,14 @@ func (c *Client) doRequest(req *http.Request, emptyResponse bool) (interface{}, 
 		var values []interface{}
 		for {
 			values = append(values, responsePaginated.Values...)
-			if len(responsePaginated.Next) == 0 {
+			if responsePaginated.Size/responsePaginated.Pagelen <= responsePaginated.Page {
 				break
 			}
 			newReq, err := http.NewRequest(req.Method, responsePaginated.Next, nil)
 			if err != nil {
 				return resBody, err
 			}
+			c.authenticateRequest(newReq)
 			resp, err := c.doRawRequest(newReq, false)
 			if err != nil {
 				return resBody, err

